docs/src/main/sphinx/connector/oracle.rst

```diff
@@ -25,16 +25,19 @@ properties in the file:
 .. code-block:: text
 
     connector.name=oracle
-    # The exact format of the connection-url varies by Oracle version. Refer to
-    # the Oracle Database documentation for version-specific information on the
-    # JDBC Thin driver.
-    connection-url=jdbc:oracle:thin:@//example.net:1521/ORCLCDB
+    # The correct syntax of the connection-url varies by Oracle version and
+    # configuration. The following example URL uses the Oracle JDBC Thin driver
```

Comment:
Reword to ensure misunderstanding .. you can NOT change the JDBC driver being used easily

Reply:
I'm conflicted on this. I see how this may imply that you can change drivers which is not advisable, but when navigating the Oracle docs for more information you probably need to know that we use the Thin driver.